hack/test-samba-container.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+LOCAL_TAG="${LOCAL_TAG:-centos8:samba}"
+
+TMPDIR="$(mktemp -d)"
+rc=$?
+
+if [ $rc -ne 0 ]; then
+	echo "Error creating temporary directory"
+else
+	CONTAINER_ID="$(podman run --network=none --name samba \
```

Comment:
do you want to change it to $CONTAINERCMD or something and allow specifying either podman or docker? I use podman by default but I think docker is still more common. Or perhaps save that for a follow up pr?

Reply:
Right, the Makefile still directly uses podman for building and tagging etc. I would want to expand that more generally to cover both. I think it can be done afterwards.

Reply:
OK, I did the docker build change first #7 ... :-)